common/djangoapps/third_party_auth/saml_configuration/tests/test_saml_configuration.py

```diff
@@ -0,0 +1,112 @@
+import unittest
+from third_party_auth.models import SAMLConfiguration
+from django.urls import reverse
+from django.contrib.sites.models import Site
+from django.contrib.auth.models import User
+
+from enterprise.constants import ENTERPRISE_ADMIN_ROLE, ENTERPRISE_LEARNER_ROLE
+from rest_framework import status
+from rest_framework.test import APITestCase
+from third_party_auth.tests import testutil
+from third_party_auth.tests.samlutils import set_jwt_cookie
+
+SAML_CONFIGURATIONS = [
+    {
+        'site': 1,
+        'slug': 'testing',
+        'private_key': 'TestingKey',
+        'public_key': 'TestingKey',
+        'entity_id': 'example.com',
+        'is_public': True
+
+    },
+    {
+        'site': 2,
+        'slug': 'testing2',
+        'private_key': 'TestingKey2',
+        'public_key': 'TestingKey2',
+        'entity_id': 'edx.example.com',
+        'is_public': True,
+    },
+]
+
+PRIV_CONFIGURATIONS = [
+    {
+        'site': 1,
+        'slug': 'testing3',
+        'private_key': 'TestingKey',
```

Comment:
perhaps does not matter, but do the site and private_key values here need to be different from the one in SAML_CONFIGURATIONS?

Reply:
No. We don't enforce uniqueness on basically anything really, but especially not those fields.